catalog-rest-service/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -320,10 +320,18 @@
         },
         "min": {
           "description": "Minimum value in a column.",
-          "type": "number"
+          "type": ["number", "integer", "string"]
         },
         "max": {
           "description": "Maximum value in a column.",
+          "type": ["number", "integer", "string"]
+        },
+        "minLength": {
```

Comment:
Hi @harshach, reverted all other changes and removed numpy dependency. The only change I'd like to stick here is: adding `minLength` and `maxLength` and updating the type for `min` and `max`. Looks like the tests are passing and this way I can compute it for numbers and strings